### PR TITLE
[Pro splash pages] Pro updates home (.en.js)

### PR DESCRIPTION
**Pull request by anothertompetty**
This PR focuses on re-doing the UK/English home page to bring in to line with Pro.

A few things have changed:

- New icons for the English home page
- New icons for the intl pages contained within `home.en.js` (e.g. Sweden)
- New sections (testimonial)
- Updated product selection section (which uses `product-grid` css)
- Created new `spotlight-grid` css since `product-grid` was being used everywhere and in lots of languages, and I only wanted to change what was on `home.en.js` for now. Could have created a new thing just for that, but went the other way round since I wanted what was used elsewhere to be more general.

Here's a screenshot:

image

This PR, and subsequent PRs are to be merged in to `pro-splash-page-updates`, which can then be merged to `dev` when we're ready to launch them.

### CODE REVIEW

**Comment by clippy** [comment]
**I have manually checked the visuals in:**

- [ ] Chrome
- [ ] IE11
- [ ] Firefox
- [ ] Safari
- [ ] iPad

**Comment by anothertompetty** [comment]
@jamesshedden ready for some reviewski-ing when you are!

**Comment by anothertompetty** [comment]
@Ken-Lo @nicola-gc let's go over the copy together when you have a chance

File: app/css/components/quote.scss [hunk 1]
```diff
@@ -0,0 +1,5 @@
+.quote {
+  background-color: $primary-color;
+  padding: 75px 175px !important; //overriding site-container padding values
```
Comment (jamesshedden) [inline app/css/components/quote.scss]: @anothertompetty Could we try just putting a new `div class="quote"` inside the `site-container` instead? Then both things should be able to do their thang

Comment (jamesshedden) [inline app/css/components/quote.scss]: (rather than adding the two classes to the same element)

Comment (anothertompetty) [inline app/css/components/quote.scss]: I think you mean `className`, JAMES

File: app/pages/home/home.en.js [hunk 2]
```diff
@@ -5,13 +5,12 @@ import Link from '../../components/link/link';
...
 import Message from '../../components/message/message';
 
-import CheckListIcon from '../../icons/svg/checklist';
-import MoneyFlowerIcon from '../../icons/svg/money-flower';
-import TickSquareIcon from '../../icons/svg/tick-square';
-import ProIcon from '../../icons/svg/pro';
-import ShopIcon from '../../icons/svg/shop';
-import IntegrationsIcon from '../../icons/svg/integrations';
-import UsersIcon from '../../icons/svg/users';
+import AutomatedyellowIcon from '../../icons/svg/automated-yellow';
```
Comment (jamesshedden) [inline app/pages/home/home.en.js]: @anothertompetty `AutomatedyellowIcon` -> `AutomatedYellowIcon` and then correct on the element as well

**Comment by jamesshedden** [comment]
@anothertompetty Top work, well done for battling with this one! Couple of quick comments, :+1: apart from that!

**Comment by anothertompetty** [comment]
Thanks a lot @jamesshedden

File: app/css/components/quote.scss (hunk 1 above)
Comment (jamesshedden) [inline app/css/components/quote.scss]: @anothertompetty Touché

**Pull request merged**